src/Common/src/CoreLib/Interop/Unix/System.Native/Interop.Stat.cs

```diff
@@ -33,6 +33,7 @@ internal struct FileStatus
             internal long BirthTimeNsec;
             internal long Dev;
             internal long Ino;
+            internal uint UserFlags;
```

Comment:
@stephentoub Should I use the new `UserFlags` enum here or leave it as `uint`?

Reply:
Let's stick with `uint`.